build.yaml

```diff
@@ -2648,6 +2676,8 @@ steps:
       - merge_code
       - memory_image
       - deploy_memory
+    clouds:
```

Comment:
Does `deploy_memory` work in Azure? Or is it just the test is too specific?

Reply:
memory might deploy but won't work because it specifically uses the GoogleStorage FS.

Reply:
Let's set gcp-only for that step as well then.